lib/main_publiccloud.pm

```diff
@@ -55,10 +55,10 @@ sub load_maintenance_publiccloud_tests {
             loadtest "xfstests/generate_report";
         } elsif (get_var('PUBLIC_CLOUD_SMOKETEST')) {
             loadtest "publiccloud/smoketest";
-            loadtest "publiccloud/flavor_check" if (is_ec2());
+            loadtest "publiccloud/flavor_check" if (is_ec2() && !check_var('PUBLIC_CLOUD_SCC_ENDPOINT', 'SUSEConnect'));
```

Comment:
I would add a comment explaining why this is excluded on that particular condition.

Reply:
done

Reply:
please confirm if text clearly explaining everything